src/retsnoop.c

```diff
@@ -1265,6 +1325,11 @@ int main(int argc, char **argv)
 		}
 	}
 
+	if (process_cu_globs()) {
+		fprintf(stderr, "Failed to process compile units.\n");
```

Comment:
how about using "file paths" terminology, instead of "compile units"? I think for most users "compile units" isn't all that meaningful (unless they are familiar with DWARF and compiler terminology)